timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/ElasticsearchClient.java

```diff
@@ -446,7 +449,7 @@ private RolloverRequest getRolloverRequest(String timbermillAlias, long maxIndex
 
 	private void updateOldAlias(RolloverResponse rolloverResponse, String timbermillAlias) throws RetriesExhaustedException {
```

Comment:
Refactor this method into sub methods, i would also consider putting it in a different class since ElasticsearchClient is becoming a 1000+ rows class, separating the logic into content relevant operations classes will make this class more readable and easier to maintain